### PR TITLE
Update event

### PR DESCRIPTION
**Pull request by petrukhinilya**

### CODE REVIEW

File: ProjectCalendar/Frontend/src/Actions/index.js [hunk 1]
```diff
@@ -59,12 +59,11 @@ export const verifyUser = (email, password) => async (dispatch) => {
 
 export const addUserEvent = (startDate, endDate, event) => async (dispatch) => {
   try {
+    const start = new Date(startDate).getTime();
+    const end = new Date(endDate).getTime();
+    let inputStartDate = Number(start);
+    let inputEndDate = Number(end);
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Actions/index.js]: Always use `const` if you won't change the variable

File: ProjectCalendar/Frontend/src/Actions/index.js [hunk 2]
```diff
@@ -135,4 +134,32 @@ export const deleteUserEvent = (id) => async (dispatch) => {
   }
 }
 
+export const updateUserEvent = (id, startDate, endDate, event) => async (dispatch) => {
+  try {
+    let inputStartDate = Number(new Date(startDate).getTime());
+    let inputEndDate = Number(new Date(endDate).getTime());
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Actions/index.js]: Always use `const` if you won't change the variable

File: ProjectCalendar/Frontend/src/Components/Calendar/Calendar.js [hunk 3]
```diff
@@ -16,6 +18,8 @@ const Calendar = () => {
   const [value, setValue] = useState(moment());
   const [showPopup, setShowPopup] = useState(false);
   const allEvents = useSelector(state => state.events.events);
+  const [changePopup, setChangePopup] = useState(false);
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/Calendar/Calendar.js]: The verb for the boolean variable?

File: ProjectCalendar/Frontend/src/Reducers/events.js [hunk 4]
```diff
@@ -27,10 +27,21 @@ const events = (state = initialEvents, action) => {
...
+            return {
+                ...state,
+                events: action.payload.events
+            }
+        case 'UPDATE_EVENTS_ERROR':
+            return {
+                ...state,
+                error: action.payload.error
+            }
+
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Reducers/events.js]: Unnecessary blank line

File: ProjectCalendar/Frontend/src/Components/Calendar/CalendarDay.js [hunk 5]
```diff
@@ -22,7 +27,7 @@ const CalendarDay = ({day, allEvents, dayStyles, setValue, deleteEvent, setChang
           {events.map((event) => {
             return (
               <div className='highlight'>
-                <div className = 'event'>{event.event}</div>
+                <div className = 'event'onClick={() => updateEvents(event)}>{event.event}</div>
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/Calendar/CalendarDay.js]: fix codestyle

**Review by d-lazarev — CHANGES_REQUESTED** [review]
Notes

File: ProjectCalendar/Frontend/src/Actions/index.js (hunk 1 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Actions/index.js]: Resolved

File: ProjectCalendar/Frontend/src/Actions/index.js (hunk 2 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Actions/index.js]: Resolved

File: ProjectCalendar/Frontend/src/Reducers/events.js (hunk 4 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Reducers/events.js]: fixed

File: ProjectCalendar/Frontend/src/Components/Calendar/CalendarDay.js (hunk 5 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/Calendar/CalendarDay.js]: Fixed

File: ProjectCalendar/Frontend/src/Components/Calendar/Calendar.js (hunk 3 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/Calendar/Calendar.js]: Fixed

File: ProjectCalendar/Frontend/src/Components/Calendar/Calendar.js [hunk 6]
```diff
@@ -18,7 +18,7 @@ const Calendar = () => {
   const [value, setValue] = useState(moment());
   const [showPopup, setShowPopup] = useState(false);
   const allEvents = useSelector(state => state.events.events);
-  const [changePopup, setChangePopup] = useState(false);
+  const [dayPopup, setDayPopup] = useState(false);
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/Calendar/Calendar.js]: How should I understand for what purpose this variable here and what it does?

File: ProjectCalendar/Frontend/src/Components/Calendar/CalendarDay.js [hunk 7]
```diff
@@ -27,7 +27,7 @@ const CalendarDay = ({day, allEvents, dayStyles, setValue, deleteEvent, setChang
           {events.map((event) => {
             return (
               <div className='highlight'>
-                <div className = 'event'onClick={() => updateEvents(event)}>{event.event}</div>
+                <div className = 'event' onClick={() => updateEvents(event)}>{event.event}</div>
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/Calendar/CalendarDay.js]: Codestyle wasn't fixed

**Review by d-lazarev — CHANGES_REQUESTED** [review]
Notes

File: ProjectCalendar/Frontend/src/Components/Calendar/Calendar.js (hunk 6 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/Calendar/Calendar.js]: Fixed

File: ProjectCalendar/Frontend/src/Components/Calendar/CalendarDay.js (hunk 7 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/Calendar/CalendarDay.js]: Fixed

**Review by d-lazarev — APPROVED** [review]

**Pull request merged**